api/v1/requests/AttendeeRequest.js

```diff
@@ -40,6 +40,10 @@ var bodyValidations = {
 	'attendee.linkedin':  ['required', 'string', 'maxLength:50'],
 	'attendee.interests': ['required', 'string', 'maxLength:255'],
 	'attendee.status': ['string', registration.verifyStatus],
+	'attendee.priority': ['required', 'integer', 'max:10', 'min:0'],
```

Comment:
How would the separate request work? Would I still be using `updateAttendee` or would I have my own method in registrationService?